modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc

```diff
@@ -7,6 +7,10 @@ This page lists new features added in Redpanda Cloud.
 
 == July 2024
 
+=== BYOC on Azure: limited availability
+
+Redpanda now supports xref:deploy:deployment-option/cloud/create-byoc-cluster-azure.adoc[BYOC clusters on Azure]. This is a limited availability release for BYOC clusters. Additional functionality is expected when this is generally available.
```

Comment:
```suggestion
Redpanda now supports xref:deploy:deployment-option/cloud/create-byoc-cluster-azure.adoc[BYOC clusters on Azure]. This is a limited availability release for BYOC clusters.
```

Reply:
The deleted text seems implicit, and we never say anything about forthcoming releases or functionality. I don't see this statement as adding value.